.cargo/config.toml

```diff
@@ -1,4 +1,7 @@
 [target.'cfg(target_os = "macos")']
 # when using osx, we need to link against some golang libraries, it did just work with this missing flags
 # from: https://github.com/golang/go/issues/42459
-rustflags = ["-C", "link-args=-framework CoreFoundation -framework Security -framework CoreServices -lresolv"]
\ No newline at end of file
+rustflags = [
+  "-C",
+  "link-args=-framework CoreFoundation -framework Security -framework CoreServices -lresolv",
+]
```

Comment:
?

Reply:
This may be changed by fmt tools, will change it back.